docs/requirements.txt

```diff
@@ -0,0 +1,7 @@
+aenum
+apache-airflow
+pydantic
```

Comment:
do we need these dependencies for doc build?

Reply:
removed